shell/platform/windows/window.cc

```diff
@@ -110,7 +110,7 @@ void Window::InitializeChild(const char* title,
   ZeroMemory(&dmi, sizeof(dmi));
   dmi.dmSize = sizeof(dmi);
   if (EnumDisplaySettings(NULL, ENUM_CURRENT_SETTINGS, &dmi)) {
-    directManipulationPollingRate_ = dmi.dmDisplayFrequency;
+    directManipulationPollingRate_ = 4 * dmi.dmDisplayFrequency;
```

Comment:
With 240Hz and this `4 * frequency`, are we assuming a 60Hz of something?

Reply:
Assuming the display is 60 Hz in case we aren't able to get the actual value. When calling `Update()` at 60 Hz I have seen 30 Hz or worse actual updates. Not sure what is the root cause, but when polling rate is increased the experience feels a lot better. At worst I saw 200ms between events. I did x4 to be safe as the velocity tracker which controls momentum only looks at the latest 100ms of events. 

I didn't spend much time on optimization here, it may be that with further testing `3 * displayFrequency` might work just as well.